src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -126,7 +126,7 @@ const StandardView = ({visToolbar, title, searchDesc, imagePlot, xyPlot, tables,
 
     const converterId = get(settingBox, ['props', 'missionEntries', LC.META_MISSION]);
     const convertData = getConverter(converterId);
-    const cutoutSize = get(convertData, 'noImageCutout') ? undefined : get(settingBox, 'props.generalEntries.cutoutSize', '0.3');
+    const cutoutSize = get(convertData, 'noImageCutout') ? undefined : get(settingBox, 'props.generalEntries.cutoutSize', '5');
```

Comment:
`cutoutSize` is used elsewhere (line 177) expecting the unit of the value in degree. The value is passed to the download component and the returned images are bigger than 5 arcminutes cutout because '5' (or any value input by user) is interpreted in the server in degree. 
You need to convert `cutoutSize` in degree before it get used in the download part.